mscore/musescore.cpp

```diff
@@ -7646,7 +7646,7 @@ MuseScoreApplication::CommandLineParseResult MuseScoreApplication::parseCommandL
       parser.addOption(QCommandLineOption({"M", "midi-operations"}, "Specify MIDI import operations file", "file"));
       parser.addOption(QCommandLineOption({"w", "no-webview"}, "No web view in start center"));
       parser.addOption(QCommandLineOption({"P", "export-score-parts"}, "Use with '-o <file>.pdf', export score and parts"));
-      parser.addOption(QCommandLineOption(      "no-fallback-font", "Don't use Bravura as fallback musical font"));
+      parser.addOption(QCommandLineOption(      "no-fallback-font", "Don't use Leland as fallback musical font"));
```

Comment:
Not sure what to make out of this, in connection with my proposal for Petaluma to fall back to MuseJazz first

Reply:
Maybe just "Don't use a fallback musical font"?

Reply:
Or leave it Bravura, as even Leland falls back to Bravura in the end

Reply:
See #6856